editor.planx.uk/src/@planx/components/shared/Preview/NavigateToPublishedButton.tsx

```diff
@@ -0,0 +1,34 @@
+import Link from "@mui/material/Link";
+import Typography from "@mui/material/Typography";
+import React from "react";
+
+import { InnerContainer } from "./SaveResumeButton";
+
+const NavigateToPublishedButton: React.FC = () => {
+  const testEnvironment = window.location.pathname.endsWith("/draft")
+    ? "/draft"
+    : "/preview";
+
+  const editorLink = window.location.pathname.replace(
+    testEnvironment,
+    "/published",
+  ) as `/${string}`;
```

Comment:
Used a type assertion here to make it clear where the slash is on this part of the link